src/components/editor/faultTree/menu/faultEvent/FaultEventMenu.tsx

```diff
@@ -376,37 +375,37 @@ const FaultEventMenu = ({ shapeToolData, onEventUpdated, refreshTree, rootIri }:
             {criticality && (
               <Typography>
                 <span className={classes.label}>{t("faultEventMenu.criticality")}:</span>
-                <span className={classes.grey}>{criticality}</span>
+                <span className={classes.selected}>{criticality}</span>
```

Comment:
This does not make sense.

"Selected" does not make sense here! We already thought about nameing such state see https://github.com/kbss-cvut/fta-fmea-ui/issues/281#issuecomment-2108228232

The reason why this is grey is because it is **read-only**.

So we need to have both classes:
- selected/nonSelected
- read-only/editable

Reply:
Added selected, nonSelected, read-only, editable styles